src/lib/Service/UserPreferences/DateTimeFormat/DateTimeFormatRepository.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Service\UserPreferences\DateTimeFormat;
+
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+
+class DateTimeFormatRepository
```

Comment:
This class name is very unfortunate. It's confusing for Doctrine developers since they associate `Repository` with Doctrine. Also Repository is how we call our kernel which makes it even more confusing.